docs/deployment.md

```diff
@@ -0,0 +1,483 @@
+KubeVirt Deployment
```

Comment:
What is the purpose of this document? Maybe worth adding it to the document itself.

Reply:
You are absolutely right.
Done.